db/schema.rb

```diff
@@ -10,57 +10,56 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2022_01_15_211102) do
-
+ActiveRecord::Schema[7.0].define(version: 2022_05_05_020054) do
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
   create_table "animal_age_preferences", force: :cascade do |t|
     t.bigint "person_id", null: false
     t.integer "animal_value"
-    t.datetime "created_at", precision: 6, null: false
```

Comment:
I'm not sure why all of these precisions went away when I generated the schema. I will take some time to investigate this